_docs/applications/mssql.md

```diff
@@ -1,36 +1,40 @@
 ---
 layout: guide
-title: StorageOS Docs - Microsoft SQL Server
+title: Microsoft SQL Server with StorageOS
 anchor: applications
 module: applications/mssql
 ---
 
 
 # ![image](/images/docs/explore/mssqllogo.png) Microsoft SQL Server 2016 with StorageOS
 
- Microsoft's latest SQL Server release represents a major update with availability for Linux platform support as well support to run as a container.
+ Microsoft's latest SQL Server release represents a major update with availability for Linux platform support and support to run as a container.
 
 ## Linux Support
 
-For Ubuntu, the Docker container requires Xenial 16.04.
+Microsoft state SQL Server for Linux runs with any supported Linux distribution.  To date we have only tested with Ubuntu 16.04 which we used for this document and Alpine Linux (which did not work).  Additional requirements include:
```

Comment:
"To date we have only tested with Ubuntu 16.04 which we used for this document and Alpine Linux (which did not work)."

Change this to "The recommended distro for testing StorageOS is Ubuntu 16.04."